bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/PersistenceResource.java

```diff
@@ -124,12 +190,12 @@ private Response getItemHistoryBean(String serviceName, String itemName, String
         }
 
         if (service == null) {
-            logger.debug("Persistence service not found '{}'.", serviceName);
+            logger.warn("Persistence service not found '{}'.", serviceName);
```

Comment:
I would prefer to keep all logging to debug only - there is nothing wrong on the server side, when invalid requests come in, so we should not clutter the log. It should rather be the client who has to deal with the error situation (and possibly provide consolidated logs through https://github.com/eclipse/smarthome/pull/759).